src/db.rs

```diff
@@ -0,0 +1,57 @@
+use sqlx::{sqlite::SqliteConnectOptions, FromRow, SqlitePool};
+use std::convert::From;
+
+#[derive(FromRow)]
+pub struct Invitation {
+    pub id: i64,
+    pub status: InvitationStatus,
+    pub code: String,
+    pub email: Option<String>,
+}
+
+pub enum InvitationStatus {
+    Pending,
+    Accepted,
+    Rejected,
+}
+
+impl From<String> for InvitationStatus {
```

Comment:
Maybe use `TryFrom` instead and don’t panic

Reply:
Tried this, but SQLx doesn't implement `TryFrom`, but only `From`, so I'll have to leave it as-is. It's not a public trait, so likely this is fine for now.